### PR TITLE
Implement dummy Http server with required endpoints

### PR DESCRIPTION
**Pull request by vogdb**
HttpServer
- /audio/publish (streamName)
- /video/publish (streamName)
- /audio/subscribe (streamName)
- /video/subscribe (streamName)

### CODE REVIEW

File: lib/cm-rest-server.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+var express = require('express');
+var http = require('http');
+var bodyParser = require('body-parser');
+var janusClient = require('janus-client');
+
+var CmRestServer = function() {
+  var app = express();
+  var router = express.Router();
```
Comment (tomaszdurka) [inline lib/cm-rest-server.js]: Maybe rather pass router as dependency and store on the instance of CmRestServer?
Also let's call it RestServer? I think CM should be reserved for classes which directly interact with CM application, wdyt?

**Comment by vogdb** [comment]
@njam FYI

**Comment by vogdb** [comment]
Currently me and Vladimir are reading about WebRTC architecture cause we've stuck with the problem of setting up of PeerConnection between Janus and Subscriber over cm-janus. The current progress is that we have a Rest server that can return an JSEP+SDP of stream by stream id. How can subscriber instantiate a video channel by that JSEP+SDP is a question at least to me.

cc @krislab @njam @tomaszdurka

**Comment by vogdb** [comment]
@kris-lab please see the comment above.

**Pull request closed without merging**